_topic_maps/_topic_map.yml

```
@@ -2834,7 +2834,7 @@ Topics:
 - Name: Workload partitioning
   File: enabling-workload-partitioning
   Distros: openshift-origin,openshift-enterprise
-- Name: Requesting CRI-O and Kubelet profiling data by using the Node Observability Operator
+- Name: Node Observability Operator
```

If you change the assembly title per my suggestion, also change it here:
```suggestion
- Name: Using the Node Observability Operator
```

---

Ack.